docs/guide/tutorials/connect/tls.txt

```diff
@@ -1,3 +1,192 @@
 =======
 TLS/SSL
 =======
+
+The Node.js driver supports TLS/SSL connections to MongoDB that support TLS/SSL support.
+
+No Certificate Validation
+-------------------------
+
+If the MongoDB instance does not perform any validation of the certificate chain, include the ``ssl=true`` in the :manual:`URI ConnectionString </reference/connection-string/>` .
+
+.. code-block:: js
+
+   const { MongoClient } = require('mongodb');
+
+   const client = new MongoClient('mongodb://localhost:27017?ssl=true');
+
+   client.connect().then(async function() {
```

Comment:
client.connect().then(async function() { should be es6 arrow fn : client.connect().then(async () => {